src/node.cpp

```diff
@@ -338,7 +335,10 @@ Node* Node::unserialize(MegaClient* client, string* d, node_vector* dp)
     hasLinkCreationTs = MemAccess::get<char>(ptr);
     ptr += sizeof(hasLinkCreationTs);
 
-    for (i = 6; i--;)
+    const auto syncableInt = MemAccess::get<int8_t>(ptr);
```

Comment:
Shouldn't nodes serialized by prior SDK versions and deserialized by this code end up with syncable == true?   I think they would end up with that field false here?  thanks

Reply:
Good point. Will fix